### ISSUE TITLE
The Legendary Address Deduplication Ticket

### ISSUE THREAD

**Issue by CowMuon** [hicommonwealth/commonwealth#3072]
The time is now.

**Comment by timolegros** [comment]
Previous ticket: #2534

**Comment by CowMuon** [comment]
Let's review the plan here and if possible execute in current sprint. Tim has finals, so first thing is for Tim Nakul & Jake to hop on a call to go over remaining steps, and if this can be completed in the current sprint. 

Also, @jnaviask is this a blocker for React launch? If so please label as such.

**Comment by NakulManchanda** [comment]
Previous ticket: https://github.com/hicommonwealth/commonwealth/issues/2534

See https://github.com/hicommonwealth/commonwealth/issues/2755 for details on first steps here, prioritized over https://github.com/hicommonwealth/commonwealth/issues/2712.

**Comment by NakulManchanda** [comment]
@MuonShot update here:
- me & @timolegros might priotize Notification/Subscription tickets instead of this

- If this should be done in parallel, we can always engage
 - @ianrowan - recent community/chain refactoring - super turnaround
- or @kurtis

Thanks!

**Comment by dillchen** [comment]
o

**Comment by jnaviask** [comment]
Superseded by #5479
